test/templates/pagination/component.njk

```diff
@@ -0,0 +1,35 @@
+{#
+    see ./data.json for object structure
+#}
+
+{% macro pagination(properties={}, modifier={}, state={}, aria={}) -%}
+<ul class="tn-pagination">
+    <li class="tn-pagination--link-wrapper">
+      <a href="#" class="tn-pagination--link" aria-label="Previous">
```

Comment:
These should be `__` classes since they are elements, modifiers use `--`. 

Use `__item` instead of `__wrapper` since it's more semantically correct. When it makes sense, use names already widely used to describe relationship, see https://www.w3.org/TR/wai-aria/roles#listitem.